src/ui/public/vislib/lib/axis_config.js

```diff
@@ -117,7 +118,7 @@ export default function AxisConfigFactory() {
     };
 
     isPercentage() {
-      return (this._values.vis._attr.mode === 'percentage');
+      return (this._chartConfig.mode === 'percentage');
```

Comment:
definetelly ... actually isPercentage() should move out of axisConfig as its not related to axis but to series (one series can be in percentage mode and the other not ... (please correct me if im wrong))